scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -213,6 +213,18 @@ class KubernetesEventsMonitor( ScalyrMonitor ):
        ```
     """
 
+    ENVIRONMENT_AWARE_CONFIG_PARAMS = {
+        "max_log_size": "SCALYR_K8S_MAX_LOG_SIZE",
+        "max_log_rotations": "SCALYR_K8S_MAX_LOG_ROTATIONS",
+        "log_flush_delay": "SCALYR_K8S_LOG_FLUSH_DELAY",
+        "message_log": "SCALYR_K8S_MESSAGE_LOG",
+        "event_object_filter": "SCALYR_K8S_EVENT_OBJECT_FILTER",
+        "leader_check_interval": "SCALYR_K8S_LEADER_CHECK_INTERVAL",
+        "leader_node": "SCALYR_K8S_LEADER_NODE",
+        "check_labels": "SCALYR_K8S_CHECK_LABELS",
+        "ignore_master": "SCALYR_K8S_IGNORE_MASTER",
+    }
+
```

Comment:
In contrast to Global and `kubernetes_monitor` params, these `kubernetes_events_monitor` param names are too generic.  So instead of relying on default prepending of `SCALYR_`,  I specify custom names.